vim25/soap/error.go

```diff
@@ -23,6 +23,8 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
We should be using the [stdlib errors](https://pkg.go.dev/errors) package instead, right?

Reply:
Thanks for the comment. I updated accordingly